src/aura/gsAppPageTemplate/gsAppPageTemplate.cmp-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<AuraDefinitionBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>49.0</apiVersion>
+    <description>Lightning Page Template used on the NPSP Getting Started page</description>
```

Comment:
```suggestion
    <description>Page Template used on the NPSP Get Started page</description>
```

Reply:
@dcaballeroSf Did you delete this file from the PR?

Reply:
This file shouldnt have to be commited. It was the templated that was required when the page was a Home page because this template was not available as a Home page. When the flexipage became a App page, there is one standard template that is the same. I think somehow the PR I closed and you talked about it to me on monday got merged into this branch by mistake. I reverted that merge

Reply:
Thanks for confirming.